client/src/django.js

```diff
@@ -51,9 +51,19 @@ const djangoClient = new Vue({
       return results;
     },
     async session(sessionId) {
+      const { data } = await apiClient.get(`/sessions/${sessionId}/deep`);
+      return data;
+    },
+    async sessionDetail(sessionId) {
```

Comment:
I'd assume that `sessionDetail` would have more information than `session`, but it seems to be the other way round.

Reply:
Detail is the right term of art in DRF-land, but I agree it's confusing next to the name `session` here. Maybe we can just name the other method `sessionDetailDeep` if we want to keep this name.

Reply:
Changed names to 
`session`: shallow request (i.e. just id, name, and lock_owner)
`sessionDeep` deep request (i.e. all above + hierarchy of experiments, scans, etc.)